tensorflow_datasets/core/features/rle_feature/rle_feature_test.py

```diff
@@ -0,0 +1,70 @@
+"""Tests for tensorflow_datasets.core.features.rle_feature."""
```

Comment:
For tests, please follow these [instructions](https://github.com/tensorflow/datasets/blob/master/CONTRIBUTING.md#tests) (some of which you've already done).